### ISSUE TITLE
User feedback: SFBOFS WL incorrect datum

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/IOOS-EDS#179]
Water Level for SFBOFS appears to not be using correct datum relationship. Comparison with CO-OPS model output on Tides and Currents shows positive MLLW values for water level, but EDS displays negative values (see Pinole Shoal as an example).
User: pat.burke@noaa.gov

**Comment by kbailey-noaa** [comment]
@Bobfrat @kknee This seems like a fairly large problem.. any thoughts?

**Comment by kbailey-noaa** [comment]
Plot comparison of SFBOFS nowcast/forecast WL vs the EDS forecast SFBOFS WLs.  Offset is ~ 6 ft.

![SFBOFS_wl](https://user-images.githubusercontent.com/11823385/58723115-457b5b00-83a7-11e9-8287-ff9405de62a2.PNG)

![EDS_SFBOFS](https://user-images.githubusercontent.com/11823385/58723127-4b713c00-83a7-11e9-9a16-821689ac0296.png)

**Comment by kbailey-noaa** [comment]
Same issue with CBOFS water levels.  It looks like the Model Viewer shows levels running several feet too low.  Offset is ~ 3 ft.

![CBOFS_wl](https://user-images.githubusercontent.com/11823385/58723656-a8212680-83a8-11e9-8b5c-4f6e18bfa80b.PNG)

![EDS_CBOFS](https://user-images.githubusercontent.com/11823385/58723662-a9eaea00-83a8-11e9-965a-70458d2e4071.png)

**Comment by Bobfrat** [comment]
@kbailey-noaa looks like we're all set now. Just checked SFBOFS and CBOFS

**Comment by kbailey-noaa** [comment]
@Bobfrat Thanks! I see it in SFBOFS now.  I'm wondering about what happens to the data before the offset adjustment made on 7/28 22:00 ET?  Are you planning on applying that offset retroactively?

**Comment by Bobfrat** [comment]
Yeah the EDS team will be applying the offset to the historic data. I'll keep you posted when that's been done.

**Comment by kbailey-noaa** [comment]
@Bobfrat CREOFS WL also not on the correct datum (in addition to GOMOFS)

**Comment by kknee** [comment]
@kbailey-noaa CREOFS is fixed. Working on GOMOFS.

I just went through each one and I think they are mostly looking good, though hoping to look at the NGOFS quickly with you

**Comment by kbailey-noaa** [comment]
@kknee Looks like CIOFS is still off. In the Viewer, see high tide near the location of NDBC buoy 46077 vs the [CO-OPS OFS plot](https://tidesandcurrents.noaa.gov/ofs/ofs_station.shtml?stname=Sehlikof%20Strait&ofs=ci&stnid=46077&subdomain=0)

**Comment by kknee** [comment]
@kbailey-noaa we dont' have vdatum grid for CIOFS yet

**Comment by kbailey-noaa** [comment]
@kknee Oh right, we discussed.  I sent a follow up to CO-OPS and added another one of their modelers.  Hopefully someone helps out soon.

**Comment by kbailey-noaa** [comment]
From Machuan: Please go to ftp site ftp://tidepool.nos.noaa.gov/pub/users/peng/outgoing/to_Kathy/ to get ciofs_vdatums.nc.

**Comment by kbailey-noaa** [comment]
@kknee @Bobfrat Possible to get CIOFS and GOMOFS datums applied before the CO-OPS demo on Thurs morning?  I know last week you mentioned you're working on it but I should've brought up the demo to inform timeline.
